platform/posix/transport/src/openssl_posix.c

```diff
@@ -662,6 +662,7 @@ OpensslStatus_t Openssl_Disconnect( const NetworkContext_t * pNetworkContext )
 }
 /*-----------------------------------------------------------*/
 
+/* coverity[misra_c_2012_rule_8_13_violation] */
```

Comment:
It is our convention to leaves some specific details right above the violation https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/master/platform/posix/transport/src/plaintext_posix.c#L111
Could you please add those?